Gruntfile.js

```diff
@@ -48,9 +48,7 @@ module.exports = function(grunt) {
           'test/lib/unittests-bundle.js': [ './test/unittests.js' ]
         },
         options: {
-          browserifyOptions: {
-            external: [ 'openpgp', 'crypto', 'node-localstorage']
-          }
+          external: [ 'openpgp']
```

Comment:
ah! now i understand why you are talking in reversed;
by the way the openpgpjs module automatically; probably yes, this line should be  "external: [ 'openpgp', 'crypto', 'node-localstorage']" but i let this to you as you have more experience in this.